src/components/game-artist/game-artist.jsx

```diff
@@ -28,23 +28,21 @@ const GameArtist = (props) => {
         <h2 className="game__title">Кто исполняет эту песню?</h2>
         <div className="game__track">
           <div className="track">
-            <button className="track__button track__button--play" type="button"></button>
-            <div className="track__status">
-              <audio src={song.src}/>
-            </div>
+            {renderPlayer(song.src, 0)}
           </div>
         </div>
 
         <form className="game__artist">
           {answers.map((answer, i) => (
             <div key={answer.artist} className="artist">
-              <input className="artist__input visually-hidden" type="radio" name="answer" value={`answer-${i}`} id={`answer-${i}`} onChange={(event) => {
-                event.preventDefault();
-                onAnswer(question, answer);
-              }}
+              <input className="artist__input visually-hidden" type="radio" name="answer" value={`answer-${i}`}
+                id={`answer-${i}`} onChange={(event) => {
+                  event.preventDefault();
+                  onAnswer(question, answer);
+                }}
```

Comment:
```suggestion
              <input 
                className="artist__input visually-hidden" 
                type="radio" 
                name="answer" 
                value={`answer-${i}`}
                id={`answer-${i}`} 
                onChange={(event) => {
                  event.preventDefault();
                  onAnswer(question, answer);
                }}
```
так легче читать пропсы